src/Maker/MakeCommand.php

```diff
@@ -58,10 +61,22 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
     {
         $command
             ->addArgument('name', InputArgument::OPTIONAL, \sprintf('Choose a command name (e.g. <fg=yellow>app:%s</>)', Str::asCommand(Str::getRandomTerm())))
+            ->addOption('old-structure', null, InputOption::VALUE_NONE, 'Generate the command with the old structure (configure and execute method) instead of an invokable command?')
```

Comment:
I think the opposite way is the good way to go, an option `invokable` would create less frictions for actual users, and we could make it opt-out  (or single behavior) in next major version.